tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -49,6 +49,12 @@ public static function prepare($scope)
 
         self::$kernel = new AppKernel('test', true);
         self::$kernel->boot();
+
+        global $kernel;
+        $kernel = self::$kernel;
+
+        $employeeId = \Employee::employeeExists('test@prestashop.com');
```

Comment:
```
/**
     * Check if Employee exists.
     *
     * @param string $email Employee email
     *
     * @return bool Indicates whether the Employee exists
     */
    public static function employeeExists($email)
```

:thinking:

Reply:
Docs are lying (once again).

Reply:
@sarjon Where is the lie? The result is casted. If you need the employee, use `getByEmail` method instead.

Reply:
Ohh, you are right.